CHANGELOG.md

```diff
@@ -5,6 +5,31 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### Removed
+* ESP-IDF v4.3 support, including mostly conditional compilations. (#420)
+### Changed
+* **ESP-IDF v4.4 is now considered deprecated.** Please start upgrading to ESP-IDF v5.
+* rmt: Symbol now derives `Clone` and `Copy` (#386)
+* GPIO20 is now included on esp32. Notice that not every esp32 has a physical GPIO20. (#370)
+* i2s: ESP-IDF v5.3 and later changed i2s_chan_config_t struct, changing a field into a unionfield and adding the new auto_clear before and after callback. (#419)
+### Added
+* reset: restart() function (#383)
+* ledc: The config_with_pin() method allows updating the LEDC's pin without destroying and recreating it. (#399)
```

Comment:
I would **not** communicate this change. It is very controversial, the author never justified "why", and in my eyes - removing this method from future releases is still very much on the table. As it "eats" pins.

Reply:
ok will remove it